xml/admin_ds_backup.xml

```diff
@@ -140,5 +136,92 @@
     </para>
    </listitem>
   </itemizedlist>
+ </sect2>
+ </sect1>
+ <sect1 xml:id="restore-ceph">
+  <title>Restoring a Ceph Node</title>
+  <para>
+   The procedure to recover a node from backup is to reinstall the node, replace
+   its configuration files, and then re-orchestrate the cluster so that the
+   replacement node is re-added.
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     Re-install the node. (See <link
+     xlink:href="https://documentation.suse.com/ses/6/single-html/ses-deployment/#ceph-install-stack"/>
+     steps 1-4.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Update to the latest patch level (See: <link
+      xlink:href="https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup-update"/>.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Install &salt; and &deepsea;. (See <link
+      xlink:href="https://documentation.suse.com/ses/6/single-html/ses-deployment/#ceph-install-stack"/>
+     steps 5-7.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Restore the old, saved &deepsea; and &salt; configuration files.
+    </para>
+    <para>
+     It may be necessary to set the permissions on the replaced
+     files. For instance:</para>
+<screen>&prompt.smaster;<command>chown</command> salt:root /etc/salt/pki/master/master.pem
+&prompt.smaster;<command>chown</command> salt:root /etc/salt/pki/master/master.pub
+&prompt.smaster;<command>chown</command> -R salt:salt /srv/pillar/ceph/</screen>
+   </listitem>
+   <listitem>
+    <para>
+     After restoring the <filename>/etc/salt</filename> directory from a backup,
+     enable and restart the &salt; services:
+    </para>
+<screen>&prompt.smaster;<command>systemctl</command> enable salt-api
+&prompt.smaster;<command>systemctl</command> start salt-api
+&prompt.smaster;<command>systemctl</command> enable salt-master
+&prompt.smaster;<command>systemctl</command> start salt-master
+&prompt.smaster;<command>systemctl</command> enable salt-minion
+&prompt.smaster;<command>systemctl</command> start salt-minion</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Register the minions against the new &salt; master. (See <xref
+     linkend="ceph-install-stack"/> steps 8-11.)
```

Comment:
How can I see the rendered output of the xref?  I suspect the "Remove the master key from all the minions" might be missing.

Reply:
You have mail.

And yes, there is no reference to this step. Can you elaborate and tell me where it should go?

Reply:
Okay, following that reference, we end http://docserv.suse.de/documents/SES_6/ses-deployment/single-html/#ceph-install-stack.  That only talks about the steps to setup the minions with a new Salt master.  Since that is a fresh deployment, it's fine.

However, after doing a recovery of a Salt master where we lost the original, all the minions still have the public key for the origianl Salt master.  Unfortunately, since Salt isn't present, these commands need to be run manually (or possibly through ssh) for every minion.

```
# rm /etc/salt/pki/minion/minion_master.pub
# systemctl restart salt-minion
```
This step could be called "Clean up the salt minions" or "Remove the old master key"  and would happen just above line 194.

Reply:
Fixed